packages/landing/src/config.map.ts

```diff
@@ -145,10 +145,14 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    if (labels == null) {
-      this.labels = layerId === 'aerial' && this.isDebug === false;
+    this.labels = false;
+
+    if (typeof labels === 'string') {
```

Comment:
the typing for `.get` is `URLSearchParam.get(name: string): string | null;`  so it can only ever be a `string` or `null`,

so `labels == null` is clearer to me over `typeof`

For example:

```typescript
new URLSearchParameters('x=1').get('x') // '1'
```